torch/_dynamo/convert_frame.py

```diff
@@ -350,20 +350,14 @@ def profile_wrapper(*args, **kwargs):
             ps.sort_stats(pstats.SortKey.TIME).print_stats(20)
             ps.sort_stats(pstats.SortKey.CUMULATIVE).print_stats(20)
 
-        maybe_upload_prof_stats_to_manifold(str(profile_path))  # fb-only
-
-        torch._logging.trace_structured(
-            "artifact",
-            lambda: {
-                "name": "dynamo_cprofile_prof",
-                "type": "prof",
-                "encoding": "base64",
-            },
-            payload_fn=lambda: base64.encodebytes(
-                open(profile_path, "rb").read()
-            ).decode("ascii"),
-        )
-
+        if manifold_link:=maybe_upload_prof_stats_to_manifold(str(profile_path)):  # fb-only
+            torch._logging.trace_structured(
+                "link",
+                lambda: {
+                    "name": "manifold_url",
```

Comment:
Let's actually call this something like "cprofile_url" or "cprofile_manifold_url", since that's how it will show up on tlparse: 
![image](https://github.com/pytorch/pytorch/assets/4811293/942fc74f-83be-4a42-bd8e-e68f971cd4fa)
(Will sort the build products too at some point)